docs/contributors/code/react-native/internationalization-guide.md

```diff
@@ -0,0 +1,75 @@
+# React Native Internationalization Guide
+
+The native version of the editor references two types of string:
+1. Strings used in web and native platforms.
+2. Strings used only in the native platform.
+
+Regarding the first type, these strings are translated following the same process described for the web version in [this guide](https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/internationalization.md), however for the latter, it's required to provide your own translations.
+
+## Extract strings only used in the native platform
+
+In order to identify these strings, you can use the script [`extract-used-strings`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/bin/extract-used-strings.js) located in `packages/react-native-editor/bin/extract-used-strings.js` to generate a JSON object that contains all the strings referenced including the platforms where they are used, as well as the files that reference it. Here you can see the format:
+```
+ {
+	"gutenberg": {
+	  "<string>": {
+	    "string": String value.
+		"stringPlural": String value with its plural form. [optional]
+		"comments": Comments for translators. [default value is an empty string]
+		"reference": Array containing the paths of the source files that reference the string.
+		"platforms": Array containing the platforms where the string is being used, values are "android" | "ios" | "web".
+	  },
+	  ...
+	},
+	"other-domain-plugin": {
+	  ...
+	},
+    ...
+}
+```
+
+This command also supports passing extra plugins, in case the React Native bundle of the editor is generated including other plugins.
+
+Filtering from this data the strings that doesn't include the "web" platform, it will provide the list of strings that are required to be translated. Once the translations for these strings are ready, they can be injected into the editor via the `translations` initial prop when it’s initialized:
```

Comment:
I'm not sure I understand the first sentence here. Would this convey the same information? 
"The command filters strings not included in the "web" platform and provides a list of strings that require translation"

Reply:
Not exactly, the command gives the full list of strings, it doesn't filter the strings not included in the "web" platform. In this case, it's up to the developer to filter the strings from the data produced by the command, translate the strings, and then implement integration to inject the translations via the initial prop. This part of the process is out of the scope of the native version, in fact, on the Gutenberg Mobile repo we covered it by providing our custom pipeline.

Reply:
@jhnstn heads up that I've rephrased this part in the following commits:
- https://github.com/WordPress/gutenberg/pull/37918/commits/199a7c825e07677fe8d808705148f1db8266c81b
- https://github.com/WordPress/gutenberg/pull/37918/commits/98db28e6613e5c659dc945d8bec2a2649e77e239

I'd appreciate it if you could take another review of the documentation, thanks 🙇 .